src/styles/_bootstrap_variables.scss

```diff
@@ -20,48 +23,80 @@ $h4-font-size: $font-size-base * 1.25 !default;
+$link-hover-color: darken($link-color, 15%) !default;
+
+// The yiq lightness value that determines when the lightness of color changes from "dark" to "light". Acceptable values are between 0 and 255.
+$yiq-contrasted-threshold: 165 !default;
 
 $navbar-dark-color: rgba(white, .5) !default;
 $navbar-light-color: rgba(black, .5) !default;
 $navbar-dark-toggler-icon-bg: url("data:image/svg+xml;charset=utf8,<svg+viewBox='0+0+30+30'+xmlns='http://www.w3.org/2000/svg'><path+stroke='#{$navbar-dark-color}'+stroke-width='2'+stroke-linecap='round'+stroke-miterlimit='10'+d='M4+7h22M4+15h22M4+23h22'/></svg>") !default;
 $navbar-light-toggler-icon-bg: url("data:image/svg+xml;charset=utf8,<svg+viewBox='0+0+30+30'+xmlns='http://www.w3.org/2000/svg'><path+stroke='#{$navbar-light-color}'+stroke-width='2'+stroke-linecap='round'+stroke-miterlimit='10'+d='M4+7h22M4+15h22M4+23h22'/></svg>") !default;
 
-$enable-shadows: true !default;
 
-$yiq-contrasted-threshold:  165 !default;
+/*** CUSTOM DSPACE VARIABLES ***/
+
+$ds-home-news-link-color: $link-color !default;
+$ds-home-news-link-hover-color: darken($ds-home-news-link-color, 15%) !default;
+$ds-breadcrumb-link-color: $link-color !default;
+
+$ds-header-navbar-border-top-color: #fff !default;
+$ds-header-navbar-border-bottom-color: #ced4da !default;
+
+$ds-header-bg: #fff !default;
+$ds-header-icon-color: $link-color !default;
+$ds-header-icon-color-hover: $link-hover-color !default;
+
+$ds-navbar-bg: #fff !default;
+$ds-navbar-dropdown-bg: #fff !default;
+$ds-expandable-navbar-bg: #fff !default;
+$ds-navbar-link-color: $link-color !default;
+$ds-navbar-link-color-hover: #{darken($ds-navbar-link-color, 15%)} !default;
+
+$ds-slider-color: darken(#94BA65, 17%) !default;
+$ds-slider-handle-color: darken(#2B4E72, 17%) !default;
 
-$primary-bg: #{lighten(map-get($theme-colors, primary), 30%)} !default;
-$secondary-bg: #{lighten(map-get($theme-colors, secondary), 30%)} !default;
-$success-bg: #{lighten(map-get($theme-colors, success), 30%)} !default;
-$info-bg: #{lighten(map-get($theme-colors, info), 30%)} !default;
-$warning-bg: #{lighten(map-get($theme-colors, warning), 30%)} !default;
-$danger-bg: #{lighten(map-get($theme-colors, danger), 30%)} !default;
+$ds-admin-sidebar-bg: darken(#2b4e72, 17%) !default;
+$ds-admin-sidebar-active-bg: darken($ds-admin-sidebar-bg, 3%) !default;
+$ds-admin-sidebar-fg: #47495d !default;
+$ds-admin-sidebar-fixed-element-width: 55px !default;
+$ds-admin-sidebar-collapsible-element-width: 250px !default;
+$ds-admin-sidebar-total-width: $ds-admin-sidebar-fixed-element-width + $ds-admin-sidebar-collapsible-element-width; // do not change/override
+
+$ds-primary-bg: #{lighten($primary, 30%)} !default;
+$ds-secondary-bg: #{lighten($secondary, 30%)} !default;
+$ds-success-bg: #{lighten($success, 30%)} !default;
+$ds-info-bg: #{lighten($info, 30%)} !default;
+$ds-warning-bg: #{lighten($warning, 30%)} !default;
+$ds-danger-bg: #{lighten($danger, 30%)} !default;
+
```

Comment:
Why have `ds` variants of these the bootstrap variables? why not simply override the original bg color variables? The way it is now, if a bootstrap component uses one of these bg color variables it will be the original bootstrap color, and when a dspace component does it it will be this lighter variant

Reply:
@davide-negretti : You appear to be referencing a commit from DSpace-CRIS.  These `$ds-*` variables do NOT exist in the DSpace codebase: https://github.com/DSpace/dspace-angular/blob/main/src/styles/_bootstrap_variables.scss

Instead, it appears the variables have a different name in DSpace: https://github.com/DSpace/dspace-angular/blob/main/src/styles/_bootstrap_variables.scss#L62-L67

So, maybe this means this PR should be updated to use the new name?

Reply:
Update: oh, I see... you renamed them and change their default settings a bit. 
For instance:
```
$primary-bg: #{lighten(map-get($theme-colors, primary), 30%)} !default;
```
Became...
```
$ds-primary-bg: #{lighten($primary, 30%)} !default;
```

Reply:
@tdonohue exactly, I linked the commit from 4science repository but it has been made by Atmire on main branch

My intent was to make clear that these are custom DSpace variables that don't exist in Bootstrap, as the previous names may have suggested

Reply:
Fair enough. Then I suggest that we simply remove the mapping to --bs css variables below to avoid that confusion. That doesn't necessarily have to happen in this PR since it doesn't add them.

Reply:
@artlowel I agree, however some of these `--bs` CSS variables are being used so they cannot be deleted. I suggest to replace their prefix with `--ds` and move them together with other `--ds` variables.

Reply:
Sounds good!